framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1793,14 +1798,28 @@ protected static void adaptGetClassReturnTypeToReceiver(final AnnotatedExecutabl
     }
 
     /**
-     * Determines the {@link AnnotatedExecutableType} of a constructor
-     * invocation. Note that this is different than calling
-     * {@link #getAnnotatedType(Tree)} or
-     * {@link #fromExpression(ExpressionTree)} on the constructor invocation;
-     * those determine the type of the <i>result</i> of invoking the
-     * constructor, which is probably an {@link AnnotatedDeclaredType}.
-     * TODO: Should the result of getAnnotatedType be the return type
-     *   from the AnnotatedExecutableType computed here?
+     * Determines the type of the invoked constructor based on the passed new
+     * class tree.
+     *
```

Comment:
Throughout, note that the formatted Javadoc will be one long paragraph unless you add explicit paragraph breaks (`<p>`).